src/main/scala/org/neo4j/spark/utils/Neo4jUtils.scala

```diff
@@ -0,0 +1,71 @@
+package org.neo4j.spark.utils
+import java.sql.Timestamp
+import java.time.{LocalDate, LocalDateTime, OffsetTime, ZoneOffset, ZonedDateTime}
+import java.util
+import java.util.concurrent.Callable
+import java.util.function
+
+import org.neo4j.driver.{Driver, Result, Session, Transaction}
+import io.github.resilience4j.retry.{Retry, RetryConfig}
+import org.apache.spark.sql.catalyst.util.DateTimeUtils
+import org.neo4j.driver.exceptions.{ServiceUnavailableException, SessionExpiredException, TransientException}
+import org.neo4j.spark.Neo4jConfig
+
+
+object Neo4jUtils {
+
+  def close(driver: Driver, session: Session): Unit = {
+    try {
+      if (session != null && session.isOpen) {
+        session.close()
```

Comment:
Please add the exception handling as we discussed.

Reply:
done